var/spack/repos/jcsda-emc-bundles/packages/jedi-tools-env/package.py

```diff
@@ -21,15 +21,16 @@ class JediToolsEnv(BundlePackage):
     # Don't install awscli and/or aws-parallelcluster via spack,
     # they are not well maintained packages and have terrible
     # dependencies. Use a venv on top of spack-stack instead.
-    #depends_on("awscli", type="run")
-    #depends_on("aws-parallelcluster", type="run")
+    # depends_on("awscli", type="run")
+    # depends_on("aws-parallelcluster", type="run")
     depends_on("py-click", type="run")
     depends_on("py-openpyxl", type="run")
     depends_on("py-pandas", type="run")
     depends_on("py-pygithub", type="run")
     depends_on("py-scipy", type="run")
     depends_on("py-sphinx", type="run")
-    depends_on("py-sphinxcontrib-bibtex", when="+latex", type="run")
+    depends_on("py-myst-parser", type="run")
```

Comment:
I took a quick look in the spack-stack configs folder and couldn't see any place where we enable/disable this variant. But you need to remove the variant entirely (lines 19 and 20).

Reply:
@climbfuji what about the mention in line 34?

Reply:
Oh, I see. You are correct, the variant needs to stay for that line. Sorry!